### PR TITLE
Hooks  renaming

### LINKED ISSUE #44: Should we rename hooks?
When I started the conan-center hook I thought it would make sense to call it the same way as the remote ``conan-center``.

Having a look at the existing hooks I would like to have the naming "somehow" unified.

This is my proposal:

- *binary-linter.py* --> *binary_linter.py*
- *bintray_update.py* --> *bintray_updater.py*
- *github-updater.py* --> *github_updater.py*

Should we rename *conan-center.py* to something like *conan-center_reviewer.py* or *conan-center_checker.py* ?

cc/ @conan-io/community-leaders

### PR DESCRIPTION
**Pull request by danimtb**
**MERGE THIS AFTER #39**

Built on top of #39 

closes #44

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,19 +37,44 @@ If you handle multiple dependencies in your project is better to add a *conan.co
...
+This hook does checks for the [inclusion guidelines of third-party libraries](https://docs.conan.io/en/latest/uploading_packages/bintray/conan_center_guide.html#inclusion-guidelines-for-third-party-libraries)
+in [Conan Center](https://bintray.com/conan/conan-center).
+
+It is mostly intended for users who want to contribute packages to Conan Center. With this hook 
+they will test some of the requirements in the guidelines, as this hook will check for recipe 
+metadata, binary matching... during the ``conan create`` step and will output the result of each 
+check as ``OK``, ``WARNING`` or ``ERROR``:
+
+```
+[HOOK - conan-center.py] pre_export(): [RECIPE METADATA] OK
```
Comment (uilianries) [inline README.md]: conan-center_reviewer.py

File: hooks/conan-center_reviewer.py [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import fnmatch
```
Comment (uilianries) [inline hooks/conan-center_reviewer.py]: Why not "conan_center_reviewer.py" ?

Comment (danimtb) [reply hooks/conan-center_reviewer.py]: As I pointed out in #44, I took conan-center as a whole concept as the name of the remote is "conan-center"

File: hooks/conan-center_reviewer.py [hunk 3]
```diff
@@ -100,6 +102,7 @@ def pre_source(output, conanfile, conanfile_path, **kwargs):
 def post_source(output, conanfile, conanfile_path, **kwargs):
     test = "[LIBCXX]"
     cpp_extensions = ["cpp", "cxx", "c++m", "cppm", "cxxm", "h++", "hh", "hxx", "hpp"]
```
Comment (uilianries) [inline hooks/conan-center_reviewer.py]: could you please add ".cc". It's the Google's C++ extension

Comment (danimtb) [reply hooks/conan-center_reviewer.py]: Opened an issue here https://github.com/conan-io/hooks/issues/49

**Pull request merged**